_posts/2019-12-16-database-migration-in-self-contained-systems.md

```diff
@@ -0,0 +1,265 @@
+---
+
+title: Database Migration in Self Contained Systems
+categories: [craft]
+modified: 2019-12-16
+author: artur
+excerpt: "A tutorial on how to migrate databases in SCS"
+image:
+  auto: 0066-database-migration
+---
+
+When you develop a application nowadays, you as programmer just want to get the database and 
```

Comment:
Try to use "we" instead of "you" throughout the text. That makes for a more conversational tone.

Reply:
done